js/modules/tamagotchi.js

```diff
@@ -1,18 +1,40 @@
 export default class Tamagotchi {
   constructor() {
-    this.health = { value: 10, importance: 1 };
-    this.hunger = { value: 10, importance: 3 };
-    this.energy = { value: 10, importance: 2 };
-    this.fun = { value: 10, importance: 4 };
+    this.health = { value: 10, importance: 1, element: null };
+    this.hunger = { value: 10, importance: 3, element: null };
+    this.energy = { value: 10, importance: 2, element: null };
+    this.fun = { value: 10, importance: 4, element: null };
     console.log("Tamagotchi initialized");
   }
 
   displayHealth = (elementSelector) => {
+    const displayElement = document.querySelector(elementSelector);
+    displayElement.innerText = this.health.value;
+  };
+
+  displayHunger = (elementSelector) => {
```

Comment:
**suggestion:** As you fixed, and set your parameter elements, you don't have to pass selector as argument now. It can be done using your class state. Eg. ` const displayElement = document.querySelector(this.hunger.element);`